ocf_data_sampler/select/space_slice_for_dataset.py

```diff
@@ -0,0 +1,53 @@
+""" Functions for selecting data around a given location """
```

Comment:
Again, nitpicky but can we rename that to `spacial_slice_for_dataset.py`? since we have `select_spacial_slice.py`